components/mitsubishi_itp/mitp_packet-get_response.cpp

```diff
@@ -99,6 +103,12 @@ float CurrentTempGetResponsePacket::get_outdoor_temp() const {
   return enhanced_raw_temp == 0 ? NAN : MITPUtils::temp_scale_a_to_deg_c(enhanced_raw_temp);
 }
 
+uint32_t CurrentTempGetResponsePacket:: get_runtime_minutes() const {
+  return pkt_.get_payload_byte(PLINDEX_RUNTIME) << 16 |
+         pkt_.get_payload_byte(PLINDEX_RUNTIME + 1) << 8 |
+         pkt_.get_payload_byte(PLINDEX_RUNTIME + 2);
+}
+
```

Comment:
Since this is under the CurrentTemp response, is this just runtime minutes for the current action?  Like minutes since it switched to actively heating?  Or is it lifetime runtime minutes?
Code is fine regardless but might be nice to have some clarification at least in comments so that we know what this is.

Reply:
It's lifetime for the unit. I have no idea why it's in the temperature report packet; I guess it's for filter/service tracking? It only increases when the unit itself is operating.